etc/schema/0.3/npq/participant_declarations/started/create/request.json

```diff
@@ -5,7 +5,7 @@
       "declaration_type": "started",
       "declaration_date": "2015-02-14T02:21:34",
       "participant_id": "0cdc3c2f-20e3-4721-97f1-485b397ae7e9",
-      "course_type": "leading-teaching"
+      "course_type": "npq-leading-teaching"
```

Comment:
I'm not sure what I'm missing, but I don't understand why this only mentions one of the 6 NPQ course types

Reply:
It's an example request to match the request_schema.json in the same directory

Reply:
Ah I see. Thanks